src/test/unit/engine/InputTest.cpp

```diff
@@ -24,297 +24,300 @@
  * Implementation of Input test
  */
 
-#include "../engine/InputTest.hpp"
 #include <nupic/engine/Input.hpp>
 #include <nupic/engine/Network.hpp>
 #include <nupic/ntypes/Dimensions.hpp>
 #include <nupic/engine/Region.hpp>
 #include <nupic/engine/Output.hpp>
+#include "gtest/gtest.h"
 
 using namespace nupic;
 
-void InputTest::RunTests()
-{
+namespace {
```

Comment:
I don't think think this is necessary / isn't commonly done so I'd remove it unless that causes some issue.